### PR TITLE
the tricksurf doc

### PR DESCRIPTION
**Pull request by HitmanVRGTAG**

### CODE REVIEW

File: 2021-12-21-guide_gamemode-info-tricksurf.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+toc_sticky: true
+---
+
+## What is Tricksurf?
+
+A spinoff mode formed from the Combat Surf game mode, players can create and complete sequences of locations to surf to, known as tricks. Some tricks are harder than others, and some can be as easy as flicking off a ramp to a certain location, therefore ending the trick. People do these tricks for XP and bragging rights.
+
+## History of Tricksurf
+
+TrickSurf Began on https://zskyworld.com/, where a Surf/Combat Surf player known as Zee began posting his servers for tricksurfing. On zskyworld people posted tricks that they had made. Later on TrickSurfing began to die, Momentum Mod is bringing it back with the full relese of there game.
```
Comment (tsa96) [inline 2021-12-21-guide_gamemode-info-tricksurf.md:26]: First off, let's keep terminology consistent, so "Tricksurf" not "TrickSurf".

Typo in "relese" an "there game" should be "their game".

Idk about saying "tricksurf began to die", seems better to be more specific that Zee's shut down. I believe that let to Tricksurf being dead for about a year, but there's `trick.surf` now. Honestly I think giving that a shoutout here would be fine. But could finish with something like " - despite that, Tricksurf remains a highly underrated mode that we hope sees more attention once added to Momentum."

Also feels like we're glossing over the history of Zee's a bit, maybe worth asking in the #general-tricksurf channel for some dates of when Zee's started and was most popular. "On zskyworld people posted tricks that they had made" reads a bit strangely - how exactly did people submit tricks, explain a little more about the system etc. Again, can always ask in the Discord, bunch of big tricksurf fans in there.

**Review by tsa96 — CHANGES_REQUESTED** [review]
Addressed a bunch of things in my one review comment. Also, this file is in the wrong place, it should be nested in the _posts/guides/ stuff.

I suggest just copying your changes somewhere outside the repo (any text editor), then doing `git reset HEAD~1 --hard` to delete this commit. Then copy your changes into the correct file, address the stuff in the review, and this time when you push, use `git push --force`. This will get rid of the bad commit with the wrong file location.

**Comment by hexaflexahexagon** [comment]
@HitmanVRGTAG Any update on addressing the above PR review?

**Comment by tsa96** [comment]
Closing due to inactivity.

**Pull request closed without merging**